front/src/modules/trainschedule/components/ManageTrainSchedule/Allowances/AllowancesStandardSettings.tsx

```diff
@@ -2,10 +2,10 @@ import React, { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import InputGroupSNCF, { InputGroupSNCFValue } from 'common/BootstrapSNCF/InputGroupSNCF';
 import OptionsSNCF from 'common/BootstrapSNCF/OptionsSNCF';
+import { onlyDigit } from 'utils/strings';
```

Comment:
Oups, there is an issue with this onlyDigit, it does not work in this file, can you remove all its usages and remove the import ? Thanks ! :)